src/glossary.rst

```diff
@@ -6301,6 +6319,15 @@ valid
 A :t:`variable` is :dt:`valid` when it has been :t:`initialized` by all
 reachable control flow paths.
 
+.. _fls_A5K8aOBsI3BG:
+
+validity invariant
+^^^^^^^^^^^^^^^^^^
+
+:dp:`fls_3ebC3l839ajF`
+A :dt:`validity invariant` is an invariant that when not uphold will result in
+immediate :t:`undefined behavior`.
```

Comment:
How about

A validity invariant is an invariant that when violated results in immediate undefined behavior.